AMW_angular/io/src/app/settings/deployment-parameter/deployment-parameter.component.ts

```diff
@@ -0,0 +1,61 @@
+import { Component, ViewChild } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
+import { ToastComponent } from '../../shared/elements/toast/toast.component';
+import { HttpClient } from '@angular/common/http';
+
+type Key = { key: string; value: any };
+
+@Component({
+  selector: 'app-deployment-parameter',
+  standalone: true,
+  imports: [CommonModule, ReactiveFormsModule, ToastComponent, FormsModule],
+  templateUrl: './deployment-parameter.component.html',
+  styleUrl: './deployment-parameter.component.scss',
+})
+export class DeploymentParameterComponent {
+  keyName = '';
+  paramKeys: Key[] = [];
```

Comment:
Consider using a observable and asyncPipe for showing new entries in the list.

1. replace the Array with an Observalbe<Key[]>
2. create a method which loads all entries (hint: same like the get request but instead of subscribe use pipe()
3. call the method from constructor
4. call the method on the successful post
5. call the method on the successful delete